### PR TITLE
Q14,Q20 done

### PR DESCRIPTION
**Pull request by VivianWenhuiLiang**

### CODE REVIEW

File: src/com/vivian/java/leetcode/Q14_LongestCommonPrefix.java [hunk 1]
```diff
@@ -5,5 +5,22 @@
  * strings.
  */
 public class Q14_LongestCommonPrefix {
-
+    public String longestCommonPrefix(String[] arr) {
+        StringBuilder sb = new StringBuilder();
+        if (arr.length == 0) {
```
Comment (GeorgeZhiXu) [inline src/com/vivian/java/leetcode/Q14_LongestCommonPrefix.java]: arr can be null as well.

File: src/com/vivian/java/leetcode/Q14_LongestCommonPrefix.java [hunk 2]
```diff
@@ -5,5 +5,22 @@
...
+            return null;
+        }
+        for (int j = 0; j < arr[0].length();) {
+            for (int i = 0; i < arr.length; i++) {
+                if (arr[i].length() < j) {
```
Comment (GeorgeZhiXu) [inline src/com/vivian/java/leetcode/Q14_LongestCommonPrefix.java]: Check the boundary case here. 
Test with data:
"abc", "abd", "ab"

File: src/com/vivian/java/leetcode/Q20_ValidParentheses.java [hunk 3]
```diff
@@ -8,5 +10,26 @@
...
 public class Q20_ValidParentheses {
+    public boolean validParentheses(String str) {
+        if (str == null) {
+            return false;
+        }
+        Stack<Character> stack = new Stack<Character>();
+        for (char c : str.toCharArray()) {
+            if (c == '(') {
+                stack.push(')');
+                continue;
```
Comment (GeorgeZhiXu) [inline src/com/vivian/java/leetcode/Q20_ValidParentheses.java]: The "continue" here is redundant
